minifi/pom.xml

```diff
@@ -504,6 +504,11 @@ limitations under the License.
                 <artifactId>lzma-java</artifactId>
                 <version>1.3</version>
             </dependency>
+            <dependency>
```

Comment:
Not specific to this line, but minifi and nifi-standard-bundle need their NOTICE files updated to include this dependency so Facebook gets the appropriate attribution since we are redistributing their binaries.

Reply:
The zstd license is already in the notice files as it is included in apache commons, which is how I'm effecting it. The jni component was either BSD or Apache license off the top of my head, also already included.

I didn't have a free rpi to test the jni component on ARM, this is my 2nd concern (the first being I can't possibly test heap utilization of every possible data flow). That's what field testing is for 😊

Reply:
Fair enough and TBH I think ARM support is still underway so anyone trying this on ARM is kinda signing themselves up to be a guinea pig in terms of support :-D